### DISCUSSION TITLE
How to understand the meaning of some resource fields

### DISCUSSION THREAD

**Discussion by wyyolo** [apache/streampipes#2483] [Q&A] [answered]
streampipes/streampipes-platform-services/src/main/java/org/apache/streampipes/ps/DataLakeResourceV4.java

![image](https://github.com/apache/streampipes/assets/105999518/76b05383-2c23-40c7-82dc-2d1edf127f56)
![image](https://github.com/apache/streampipes/assets/105999518/e13318ae-0ed0-4932-a9ab-21b7cc0e0ae8)
Could you please tell me what "measurementID" and "measurmentId" mean?

And,
![157a9c9803854d9ec7c35b54abf5916](https://github.com/apache/streampipes/assets/105999518/98d45699-b64b-4113-ac95-937dd43ea49c)
 In this path :/measurements ，I can obtain the measurement using the GET method, but using Delete cannot delete the measurement and returned an error code of 500.

**Comment by dominikriemer** [comment, marked as answer]
Hi @wy-os a bit ugly, but the `measurementID` refers to the `measureName` in the `DataLakeMeasure` object, which contains the metadata for the stored InfluxDB measurement. The measureName is the actual name of the measurement in Influx.

It would probably be better to support the document id of the `DataLakeMeasure` in the API, but we need to check if this breaks anything.

Please ask if it doesn't answer your question!
